src/screens/common/RegisterEvent/RegisterEvent.tsx

```diff
@@ -27,22 +28,25 @@ const RegisterEvent: React.FC<RegisterEvent> = ({ eventId, userId }) => {
   } = useForm();
   const [
     createInvitation,
-    { data, error, loading },
```

Comment:
Maybe when the user registers, we could show a success snackbar that says "Successfully registered"